src/analyses/is_threaded.cpp

```diff
@@ -58,30 +50,31 @@ class is_threaded_domaint:public ai_domain_baset
     ai_baset &ai,
     const namespacet &ns) final
   {
+    // assert(reachable);
+
```

Comment:
Once goto-analyzer is merged this assertion will hold and I plan to then remove the if statements in ``transform()`` and ``merge()`` and uncomment the assertions.